--- third_party/fiat/curve25519_64_adx.h	2023-10-19 12:36:31.082366700 -0700
+++ third_party/fiat/curve25519_64_adx.h.mingw	2023-10-19 12:34:52.467861300 -0700
@@ -35,7 +35,7 @@
 __attribute__((target("adx,bmi2")))
 static inline void fiat_mulx_u64(uint64_t* out1, uint64_t* out2, uint64_t arg1, uint64_t arg2) {
 // NOTE: edited after generation
-#if defined(_M_X64)
+#if defined(_M_X64) && !defined(__MINGW32__)
   unsigned long long t;
   *out1 = _umul128(arg1, arg2, &t);
   *out2 = t;
